app/server.go

```diff
@@ -138,6 +138,7 @@ type Server struct {
 
 func NewServer(options ...Option) (*Server, error) {
 	rootRouter := mux.NewRouter()
+	rootRouter.SkipClean(true)
```

Comment:
I'm not 100% sure if this change is backward compatible.

Reply:
An ugly fix for backward compatibility would be copying the cleaning code from `mux` and putting it into a middleware that matches all other routes except the email one. But it'd be very ugly.

Reply:
After discussing with @jupenur offline, we've decided to keep the fix contained and choose backwards compatibility over solving the unlikely case of emails with quotes and `..`, so I've reverted the commit that applied this change